Serverless/Azure Functions/In-process and isolated functions.md

```diff
@@ -1,20 +1,19 @@
+Functions support two process models for .NET class library functions:
+
 * In-Process - Azure Functions host is the runtime for .NET functions. Before .NET 5 C# function apps run in the same process as the host. Sharing a process has enabled unique benefits to .NET functions, most notably a set of rich bindings and SDK injections. However, as a side effect, dependencies could conflict (like `Newtonsoft.Json`) and running in the same process also means that the .NET version of the user code must match the .NET version of the host.
-* Out Of Process (Isolated) - Out of process is running functions as an isolated process from the main Azure Functions host. This allows Azure Functions to rapidly support new versions of these languages without updating the host, and it allows the host to evolve independently to enable new features and update its own dependencies over time.
+* Out Of Process (Isolated) - Out of process is running functions as an isolated process from the main Azure Functions host. This allows Azure Functions to rapidly support new versions of these languages without updating the host, and it allows the host to evolve independently to enable new features and update its dependencies over time.
 
-Differences:
+The two different process models allow for different configurations and features:
 
 * Dependency injection for out-of-process functions is done via the usual .NET core pattern while in-process functions need a custom StartUp class.
```

Comment:
This one is from before, but I believe that instead of `StartUp` it should be  `Startup`, without the pascalcase-like casing 😄. Also, since it is a name of the class, I think we should wrap it with the code element.